guides/source/developers/products-and-variants/product-images.html.md

```diff
@@ -50,24 +50,19 @@ You can check the default settings by calling the `attachment_definitions`
 method on `Spree::Image` in your Rails console:
 
 ```ruby
-Spree::Image.attachment_definitions[:attachment][:styles]
-=> {
-  mini=>"48x48>",
-  small=>"100x100>",
-  product=>"240x240>",
-  large=>"600x600>"
-}
+>> Spree::Image.attachment_definitions[:attachment][:styles]
+=> {:mini=>"48x48>", :small=>"400x400>", :product=>"680x680>", :large=>"1200x1200>"}
 ```
 
 The default sizes can be changed in an initializer. For example, in your
 `config/initializers/paperclip.rb` file. You can set new defaults like this:
```

Comment:
Would it make sense to make it super clear that those were the old defaults? For example by changing the wording to something like:
> You can set new defaults like this (these were the default values for Solidus up to version 2.9.x):

Reply:
@spaghetticode that's a great suggestion, it makes total sense for me. @elia thoughts?